redisinsight/ui/src/packages/redistimeseries-app/src/components/Chart/Chart.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { useRef, useEffect } from 'react'
+import Plotly, { Layout } from 'plotly.js-dist-min'
+import { Legend, LayoutAxis, PlotData } from 'plotly.js'
+import moment from 'moment'
```

Comment:
moment.js is depricated
Could you replace it to [date-fns](https://date-fns.org/)?